### PR TITLE
feat(netrc): add support for getting github credentials from the .netrc ...

### PR DESCRIPTION
**Pull request by frankwallis**
...file

This changes uses the 'node-netrc' package to read github credentials from ~/.netrc. The netrc file is only used if the credentials have not been configured in ~/.jspm/config (via the jspm cli)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "decompress-zip": "^0.1.0",
     "graceful-fs": "^3.0.6",
     "mkdirp": "~0.5.0",
+    "node-netrc": "0.0.1",
```
Comment (guybedford) [inline package.json]: We should probably just use https://www.npmjs.com/package/netrc directly?

**Comment by guybedford** [comment]
This is really great, thanks!

File: package.json (hunk 1 above)
Comment (frankwallis) [inline package.json]: Agreed, I'll make the change

**Comment by frankwallis** [comment]
closing, will reapply in separate PR

**Pull request closed without merging**